core/src/validator.rs

```diff
@@ -387,6 +387,10 @@ impl Validator {
             &start_progress,
         );
 
+        if !config.no_poh_speed_test {
```

Comment:
We could do it even sooner, in `new_banks_from_ledger`, immediately after loading the genesis config

Reply:
Good call! Done